modules/claims_api/app/controllers/claims_api/v0/claims_controller.rb

```diff
@@ -14,15 +14,24 @@ def index
         render json: claims,
                serializer: ActiveModel::Serializer::CollectionSerializer,
                each_serializer: ClaimsApi::ClaimListSerializer
+      rescue EVSS::ErrorMiddleware::EVSSError
```

Comment:
i would log these as warnings

Reply:
I'm down with doing that. Seems like it could be useful in debugging in case that issue arises. Will add.